server/modules/category/service/categoryService.js

```diff
@@ -26,8 +89,12 @@ export const {
   countCategories,
   fetchCategoriesBySurveyId,
   fetchCategoriesAndLevelsBySurveyId,
+  fetchCategoryUuid,
   fetchCategoryAndLevelsByUuid,
+  fetchLevelsByCategoryUuid,
+  fetchItemsByCategoryUuid,
   fetchItemsByParentUuid,
+  fetchCategoryCodesListStream,
```

Comment:
check where they are used and remove the unnecessary ones

Reply:
Solved!